first draft of introduction and customize-homepage (#1)
thibaudcolas/tutorial-series: customize-homepage.md

        "wagtailimages.Image",
        null=True,
        blank=True,
        on_delete=models.SET_NULL,
        related_name="+",
        help_text="Homepage image",
    )
    hero_text = models.CharField(
        blank=True,
        max_length=255, help_text="Write an introduction for the site"
    )
    hero_cta = models.CharField(
        blank=True,
        verbose_name="Hero CTA",
        max_length=255,
        help_text="Text to display on Call to Action",
    )
    hero_cta_link = models.ForeignKey(
        "wagtailcore.Page",
        null=True,
        blank=True,
        on_delete=models.SET_NULL,
        related_name="+",
        verbose_name="Hero CTA link",
        help_text="Choose a page to link to for the Call to Action",
    )

    # Body section of the HomePage
    body = RichTextField(blank=True)

    content_panels = Page.content_panels + [
        MultiFieldPanel(
            [
                FieldPanel("image"),
                FieldPanel("hero_text"),
                FieldPanel("hero_cta"),
                FieldPanel("hero_cta_link"),
            ],
            heading="Hero section",
        ),
        FieldPanel('body'),
    ]
```

> You might already be familiar with the different parts of your Homepage model. The `image` field is a `ForeignKey` referencing Wagtail's built-in Image model for storing images. Similarly, `hero_cta_link` is a `ForeignKey` to `wagtailcore.Page`, which is a page instance. 

Author: `wagtailcore.Page` is a class, not an instance. It’s the base class for all other page types in Wagtail (as in – all inherit from it). For example – `class BlogPage(Page)` – that `Page` is `wagtailcore.Page`.

We import it as `from wagtail.models import Page` because it’s where it is situated in the code - and that part of the code is registered in Django as the "wagtailcore" app.

```suggestion
Similarly, `hero_cta_link` is a `ForeignKey` to `wagtailcore.Page`, which is the base class for all page types, therefore allowing us to choose a page of any type.
```

> Using `on_delete=models.SET_NULL` ensures that if you remove an image or hero link from your admin interface, the `image` and `hero_cta_link` fields on your Homepage will be set to null, preserving their entries. 

Author: This is good. If you want you could link to Django docs about on_delete so people can learn about other options.

> The `related_name` attribute defines the reverse relationship from the related model, providing a readable attribute name when accessing associated objects. For instance, if you want to access `image` from `wagtailimages.Image`, you can use the value that you give to the `related-name` attribute in your QuerySet. When you use `related_name="+"`, then you don't want to create a reverse relationship for your `ForeignKey` fields. In other words, you're instructing Django not to create a way to access `image` from `wagtailimages.Image` and `hero_cta_link` from `wagtailcore.Page`. 

Author: Just specifying the model’s name so it’s clear we‘re referring to its field.

```suggestion
The `related_name` attribute defines the reverse relationship from the related model, providing a readable attribute name when accessing associated objects. For instance, if you want to access the HomePage’s `image` from `wagtailimages.Image`, you can use the value that you give to the `related_name` attribute in your QuerySet. When you use `related_name="+"`, then you don't want to create a reverse relationship for your `ForeignKey` fields. In other words, you're instructing Django not to create a way to access the HomePage’s `image` from `wagtailimages.Image` and `hero_cta_link` from `wagtailcore.Page`. 
```

While `body` is a `RichTextField`, `hero_text` and `hero_cta` are `CharField`, a typical Django string field for storing short texts.

> The [Your First Wagtail Tutorial]() already explained `content_panels`. [FieldPanel](https://docs.wagtail.org/en/stable/reference/pages/panels.html#fieldpanel) and [MultiPanel](https://docs.wagtail.org/en/stable/reference/pages/panels.html#multifieldpanel) are types of Wagtail built-in [Panels](https://docs.wagtail.org/en/stable/reference/pages/panels.html#panel-types). They are both subclasses of the base Panel class and accept all of Wagtail's `Panel` parameters in addition to their own. While the `FieldPanel` provides a widget for basic Django model fields, `MultiPanel` helps group related fields.

Author: Typo & suggest rewriting a bit to explain the point of MultiFieldPanel is to help people decide what the best structure of their form is.

```suggestion
The [Your First Wagtail Tutorial]() already explained `content_panels`. [FieldPanel](https://docs.wagtail.org/en/stable/reference/pages/panels.html#fieldpanel) and [MultiPanel](https://docs.wagtail.org/en/stable/reference/pages/panels.html#multifieldpanel) are types of Wagtail built-in [Panels](https://docs.wagtail.org/en/stable/reference/pages/panels.html#panel-types). They are both subclasses of the base Panel class and accept all of Wagtail's `Panel` parameters in addition to their own. While the `FieldPanel` provides a widget for basic Django model fields, `MultiFieldPanel` helps you decide the structure of the editing form. For example, you can group related fields.
```
